EssentialFeed/EssentialFeediOS/Image Comments UI/Controllers/ImageCommentCellController.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public final class ImageCommentCellController: NSObject {
+	private let viewModel: ImageCommentViewModel
+	private var cell: ImageCommentCell?
+
+	public init(viewModel: ImageCommentViewModel) {
+		self.viewModel = viewModel
+	}
+}
+
+extension ImageCommentCellController: UITableViewDataSource {
+	public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		1
+	}
+
+	public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		cell = tableView.dequeueReusableCell()
+
+		cell?.authorLabel.text = viewModel.author
+		cell?.dateLabel.text = viewModel.date
+		cell?.commentLabel.text = viewModel.comment
+
+		return cell!
```

Comment:
Following my previous message, this class doesn't need to hold a strong reference to the cell. It can just create the cell and return it - `UITableView` will hold a strong reference to it already as a subview:

```suggestion
		let cell: ImageCommentCell = tableView.dequeueReusableCell()
		cell.authorLabel.text = viewModel.username
		cell.dateLabel.text = viewModel.date
		cell.commentLabel.text = viewModel.message
		return cell
```